pp/integrations/plugin-packs/procedures/operatingsystems-linux-snmpv3.md

```diff
@@ -372,18 +365,18 @@ yum install centreon-plugin-Operatingsystems-Linux-Snmp
 4. Fill in the macros you want. Some macros are mandatory.
 
-| Macro                | Description                                                                                                                                                          | Default value     | Mandatory   |
-|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| SNMPV3USERNAME       | SNMP v3 only: User name (securityName)                                                                                                                               |                   |             |
-| SNMPV3AUTHPROTOCOL   | SNMP v3 only: Authentication protocol: MD5\|SHA. Since net-snmp 5.9.1: SHA224\|SHA256\|SHA384\|SHA512                                                                |                   |             |
-| SNMPV3PRIVPROTOCOL   | SNMP v3 only: Privacy protocol (privProtocol) used to encrypt messages. Supported protocols are: DES\|AES and since net-snmp 5.9.1: AES192\|AES192C\|AES256\|AES256C |                   |             |
-| SNMPV3AUTHPASSPHRASE | SNMP v3 only: Pass phrase hashed using the authentication protocol defined in the --authprotocol option                                                              |                   |             |
-| SNMPV3PRIVPASSPHRASE | SNMP v3 only: Privacy pass phrase (privPassword) to encrypt messages using the protocol defined in the --privprotocol option                                         |                   |             |
-| SNMPEXTRAOPTIONS     | Any extra option you may want to add to every command (a --verbose flag for example). All options are listed [here](#available-options).                                                                 |                   |             |
+| Macro                | Description                                                                                                                              | Default value                         | Mandatory   |
+|:---------------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| SNMPV3USERNAME       | SNMP v3 only: User name (securityName)                                                                                                   |                                       |             |
+| SNMPV3AUTHPROTOCOL   | SNMP v3 only: Authentication protocol: MD5\| SHA. Since net-snmp 5.9.1: SHA224\|SHA256\|SHA384\|SHA512                                                                |                   |             |
```

Comment:
```suggestion
| SNMPV3AUTHPROTOCOL   | SNMP v3 only: Authentication protocol: MD5\|SHA. Since net-snmp 5.9.1: SHA224\|SHA256\|SHA384\|SHA512                                                                |                   |             |
```